src/runtime.cpp

```diff
@@ -1208,6 +1208,20 @@ namespace hpx
         return rt->get_thread_manager().get_worker_thread_num();
```

Comment:
Sure, good point. My rationale for the new function was to leave the API unchanged.